apps/console/src/features/users/components/guests/pages/invite-parent-org-user.tsx

```diff
@@ -156,6 +159,32 @@ export const InviteParentOrgUser: FunctionComponent<InviteParentOrgUserPropsInte
                                     />
                                 ))
                             }
+                            filterOptions={ (_options: any, params: any) => {
```

Comment:
refrain from using `any` if possible

Reply:
Couldn't find a proper type for options and params. Therefore, kept it as it is. 

Anyways, updated the types in the `renderOption` method signature.

Reply:
ack